src/patternfly/components/Table/examples/table-expandable-example.hbs

```diff
@@ -1,4 +1,4 @@
-{{#> table table--grid="true" table--modifier="pf-u-box-shadow-md pf-m-grid-lg" table--expandable="true" table--attribute='aria-label="Expandable table example"'}}
```

Comment:
this removes the box-shadow, which is intentional right? This seems to align with the design.

Reply:
yep!